app/actions/EventActions.js

```diff
@@ -386,13 +386,17 @@ export function unfollow(followId: number, eventId: number): Thunk<Promise<*>> {
     );
 }
 
-export function isUserFollowing(eventId: number, userId: number) {
-  return callAPI({
-    types: Event.IS_USER_FOLLOWING,
-    endpoint: `/followers-event/?target=${eventId}&follower=${userId}`,
-    method: 'GET',
-    meta: {
-      errorMessage: 'Henting av hendelse følgere feilet'
-    }
-  });
+export function isUserFollowing(eventId: number): Thunk<Promise<*>> {
+  return (dispatch, getState) =>
+    dispatch(
+      callAPI({
+        types: Event.IS_USER_FOLLOWING,
+        endpoint: `/followers-event/?target=${eventId}&follower=${getState()
+          .auth.id || ''}`,
+        method: 'GET',
+        meta: {
+          errorMessage: 'Henting av hendelse følgere feilet'
```

Comment:
I don't really like the phrasing of this error message

Reply:
I know it was there before your edit, but this is a prime time to change it :)

Reply:
Yeah, I noticed when testing after I committed, sounds like a quick copy-paste-adapt gone a bit wrong

Reply:
Way better to write 3 comments instead of fixing it?? :thinking: